README.md

```diff
@@ -86,6 +88,18 @@ Link
 
 `<%= link_to 'Login with Census', '/auth/census' %>`
 
+#### Step 6: Force SSL
+
+Census requires you to use a secure callback URL when registering your application. In order to ensure that your application uses a secure connection, add the line below to your `config/application.rb` file within the Application class definition.
+
+```
+config.force_ssl = true
+```
+
+In order for your application to work locally, you will need to generate an SSL certificate. For more information on that see the links below.
```

Comment:
I don't need to setup ssl locally to log into apps. What issues did you run into without this change locally?

Reply:
Chrome gives an error saying that `This site can’t provide a secure connection`.

My server log produces the message below:

```
2018-07-29 09:57:32 -0600: HTTP parse error, malformed request (): #<Puma::HttpParserError: Invalid HTTP format, parsing fails.>
```

Reply:
Both of these are due to the fact that I'm using `https` locally because Census told me that it required it when I tried to provide a callback URL starting with `http`.